pkgs/by-name/vp/vpd/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitiles
+, libuuid
+, makeWrapper
+, flashrom
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vpd";
+  version = "R117-15572.B";
```

```suggestion
  version = "127-15917.B";
```

a bump, we strip the `R`, otherwise `lib.getVersion vpd.name` will misbehave